h2o-core/src/test/java/water/api/CloudHandlerTest.java

```diff
@@ -0,0 +1,25 @@
+package water.api;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.TestUtil;
+import water.api.schemas3.CloudV3;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CloudHandlerTest extends TestUtil {
+
+  @BeforeClass
+  public static void beforeClass() {
+    TestUtil.stall_till_cloudsize(1);
+  }
+
+  @Test
+  public void status() {
+    final CloudHandler cloudHandler = new CloudHandler();
+    final CloudV3 status = cloudHandler.status(3, new CloudV3());
+    assertNotNull(status);
+    assertNotEquals(status.leader_idx, -1);
```

Comment:
it would be nice to send a request to establish a leader and then do another where the return value would be >= 0

Reply:
Good idea! In this tests we already have leader node assigned and test that the value is >=0

Reply:
oh true, I missed the Not in assertNotEquals